### PR TITLE
new mapping stuff

### PR DESCRIPTION
**Pull request by Blayyke**

### CODE REVIEW

**Comment by Blayyke** [comment]
fixes yaw/pitch being mapped the wrong way round too

File: mappings/net/minecraft/client/render/entity/TileEntityRenderDispatcher.mapping [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CLASS ll net/minecraft/client/render/entity/TileEntityRenderDispatcher
+	FIELD a INSTNACE Lll;
+	METHOD a renderBlockEntity (Low;DDDF)V
```
Comment (valoeghese) [inline mappings/net/minecraft/client/render/entity/TileEntityRenderDispatcher.mapping]: `renderTileEntity`

File: mappings/net/minecraft/entity/FallingBlock.mapping [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CLASS ju net/minecraft/entity/FallingBlock
```
Comment (valoeghese) [inline mappings/net/minecraft/entity/FallingBlock.mapping]: `FallingTile`

File: mappings/net/minecraft/entity/player/Player.mapping [hunk 3]
```diff
@@ -5,12 +5,29 @@ CLASS gs net/minecraft/entity/player/Player
...
+	FIELD j handSwinging Z
+	FIELD k handSwingTicks I
 	FIELD m dimensionId I
 	FIELD u sleeping Z
+	FIELD v bedPosition Lbr;
+	METHOD D dropSelectedItem ()V
+	METHOD E updateEyeHeight ()V
 	METHOD G getHeldItem ()Liz;
+	METHOD J swingHand ()V
+	METHOD O hasSleptEnough ()Z
```
Comment (valoeghese) [inline mappings/net/minecraft/entity/player/Player.mapping]: I wonder why this is a method in beta 1.7.3... it isn't used for anything is it?

File: mappings/net/minecraft/util/io/CompoundTag.mapping [hunk 4]
```diff
@@ -52,7 +52,7 @@ CLASS nu net/minecraft/util/io/CompoundTag
 		ARG 1 key
 	METHOD j getByteArray (Ljava/lang/String;)[B
 		ARG 1 key
-	METHOD k getCompoundTag (Ljava/lang/String;)Lnu;
+	METHOD k getCompound (Ljava/lang/String;)Lnu;
```
Comment (valoeghese) [inline mappings/net/minecraft/util/io/CompoundTag.mapping]: ok.. I'm interested as to why you made this change

File: mappings/net/minecraft/entity/player/Player.mapping (hunk 3 above)
Comment (Blayyke) [reply mappings/net/minecraft/entity/player/Player.mapping]: sleeping?

File: mappings/net/minecraft/util/io/CompoundTag.mapping (hunk 4 above)
Comment (Blayyke) [reply mappings/net/minecraft/util/io/CompoundTag.mapping]: Fixed.

File: mappings/net/minecraft/entity/FallingBlock.mapping (hunk 2 above)
Comment (Blayyke) [reply mappings/net/minecraft/entity/FallingBlock.mapping]: Fixed.

File: mappings/net/minecraft/client/render/entity/TileEntityRenderDispatcher.mapping (hunk 1 above)
Comment (Blayyke) [reply mappings/net/minecraft/client/render/entity/TileEntityRenderDispatcher.mapping]: Fixed.

**Pull request merged**